src/dataflow/models/tools/dataflow/dataflow-content.ts

```diff
@@ -0,0 +1,21 @@
+import { types, Instance } from "mobx-state-tree";
+
+export const kDataflowToolID = "Dataflow";
+
+export function defaultDataflowContent(): DataflowContentModelType {
+  return DataflowContentModel.create();
+}
+
+export const kDataflowDefaultHeight = 480;
```

Comment:
480 might be a little short.  However, the program editor div is also constrained to 500px at present.  In `dataflow-program.sass` we would also need to adjust the height of the `flow-tool` class.

Reply:
Ultimately, we'll probably want some sort of use-all-available-space setting, but that's a story for another day.